Umbraco-Cloud/Troubleshooting/Log-files/index.md

```diff
@@ -36,6 +36,20 @@ When you're in Kudu, you can go up to your `site` folder as described in the 5 s
 
 - `log.log` shows you the same output you will have seen when pushing your changes using Git, it will show you what happened during the push and if any errors occurred. This file is especially useful when trying to find errors for deploys using the portal (so from dev > live or from dev > staging > live). Even though the last line may end with "Deployment successful" it is possible that there were errors or suspicious messages before that so make sure to give them a read.
 
+## Cleaning up the environment logs viewer
+
+The logs viewer can sometimes timeout indifinitely due to too many logs which makes it impossible to view and read them.
```

Comment:
```suggestion
On Cloud environments all errors are logged to a database table, which is what is seen in the portal under each environment. If you leave too many unread log messages it can cause timeouts when you go to see your errors. 

Since these errors are stored in your database you can go clean them up by [accessing the environment database](../../Databases/Cloud-Database).
```

I think something like this sounds a bit nicer, saying it can "sometimes timeout indefinitely" is a bit harsh, and makes it seem like it can happen at random - when in reality it only really happens if you have crazy amount of unread log messages 😉